mkt/extensions/models.py

```diff
@@ -267,6 +267,7 @@ def mini_manifest(self):
         except ExtensionVersion.DoesNotExist:
             return {}
         mini_manifest = {
+            'id': self.uuid,
```

Comment:
I don't think the mini-manifest for add-ons is documented anywhere, except on MDN where it basically says "this is a webapp manifest" :( It's really an internal thing.